spark_client/resources/conf/spark-defaults.conf

```diff
@@ -0,0 +1 @@
+spark.kubernetes.container.image=docker.io/averma32/sparkrock:latest
```

Comment:
I'd like to avoid merging this PR with a non-Canonical container repo in the config

Reply:
Yes, we should definitely do this before publishing the SNAP. It would probably really take minimal efforts do upload the image to the dataplatformoci account